src/redux/books/books.js

```diff
@@ -0,0 +1,76 @@
+import { v4 as uuidv4 } from 'uuid';
+
+// Initial book state
+const bookState = [
+  {
+    id: 1,
+    title: 'Toilet Paper',
+    category: 'Science',
+    author: 'Frank dejoin',
+    status: true,
+  },
+  {
+    id: 2,
+    title: 'Melborn Syndy',
+    category: 'Clothing',
+    author: 'Esther Adeleke',
+    status: false,
+  },
+  {
+    id: 3,
+    title: 'Metal Chair',
+    category: 'Adame',
+    author: 'Sule Marke',
+    status: false,
+  },
+
+  {
+    id: 4,
+    title: 'Daniel Ufeli',
+    category: 'Agent47',
+    author: 'Onos Mark',
+    status: true,
+  },
+];
+
+// create action contants - This are calleds action types
+const ADD_BOOK = 'books/ADD_BOOK';
+const REMOVE_BOOK = 'books/REMOVE_BOOK';
```

Comment:
- [ ] Kindly be noticed that the value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: `const ADD_BOOK = bookstore/books/ADD_BOOK` and `const REMOVE_BOOK = bookstore/books/REMOVE_BOOK`.